be/src/vec/core/sort_block.cpp

```diff
@@ -114,6 +114,60 @@ void sort_block(Block& block, const SortDescription& description, UInt64 limit)
     }
 }
 
+void sort_block(Block& src_block, Block& dest_block, const SortDescription& description,
```

Comment:
here many code is same as upper， why only one method， you can call same `src_block` and `dest_block` to replace upper method